src/lib/openapi/spec/tag-types-schema.ts

```diff
@@ -6,12 +6,18 @@ export const tagTypesSchema = {
     type: 'object',
     additionalProperties: false,
     required: ['version', 'tagTypes'],
+    description:
+        'A list of tag types with a version number representing the schema used to model the tag types.',
     properties: {
         version: {
             type: 'integer',
+            description:
+                'The version of the schema used to model the tag types.',
+            example: 1,
```

Comment:
How many versions do we have? Could we maybe use an enum to show all the versions here?

Reply:
Only 1 AFAIK, and it will probably never change. This is reserved for major changes, but I don't think modeling a version as an enum is a good idea as adding a new version would be a breaking change for older clients (which will not be gracefully handled)

Reply:
Hmm, I see your reasoning here. However, I feel like giving users a clear description of what values we expect to deliver would be also be good. Maybe an enum and stating explicitly that more versions can be added in the future, but that any breaking changes would go in a major version? 

If not, I'm happy to leave it as is too.

Reply:
> Maybe an enum and stating explicitly that more versions can be added in the future

I think the version being an integer states exactly that

Reply:
Talking about it with @chriswk there might be some benefits of having it as an enum. I did some digging and found [this](https://stackoverflow.com/questions/50760778/how-to-do-version-control-with-enums) and [this](https://softwareengineering.stackexchange.com/questions/345773/semantic-versioning-in-openapi-with-string-enum-new-values-ok), none of those were conclusive. I asked ChatGPT and this is what I got:

> Including a version field in the response of an OpenAPI specification can be a good idea in certain situations, especially when dealing with APIs that are subject to frequent updates or have different versions coexisting. However, whether to represent the version field as an enum or a string/integer depends on your specific use case and requirements. Let's discuss the considerations for each option:
> 
> 1. Enum:
> Using an enum to represent the version field in the response has some benefits. It provides a predefined set of allowed values, ensuring that the version is always one of the specified options. This can help with consistency, validation, and avoiding potential typos or accidental variations in the version representation.
> 
> However, using an enum for versioning can also introduce some challenges. Enums are typically used for representing a finite set of known values, assuming that the set won't change frequently. If your API has a dynamic versioning system with a large number of potential versions, it may be impractical to maintain an enum that includes all possible values. Additionally, if the versioning system allows for arbitrary or user-defined values, an enum may not be suitable.
> 
> 2. String or Integer:
> Representing the version field as a string or an integer provides more flexibility compared to an enum. It allows for arbitrary version values, including custom or vendor-specific formats. This flexibility can be advantageous when dealing with complex versioning schemes or when the API consumers need to parse and interpret the version information in a specific way.
> 
> Using a string or integer for the version field also makes it easier to add new versions without modifying the API specification. You can simply introduce a new value without needing to update the enum definition or regenerate client code.
> 
> Ultimately, the decision between using an enum, string, or integer for the version field depends on the specific requirements of your API and how you want to manage versioning. Consider factors such as the expected number of versions, whether the set of versions is known and stable, and how the version information will be used by API consumers.

Overall, this doesn't seem like a decision to take on lightly in this particular API. I think this is worth some general alignment and changing this across the board. 

Most of our versions are numbers, a few are enums and there's no consistency on how we define versions. I think this should be our focus, but for now, I'd like to keep it as an integer which is what majority of our APIs are using

Reply:
Yeah, I agree. Thanks for flagging that and having the discussion! Let's figure it out later 😄